src/kontrol/solc_to_k.py

```diff
@@ -329,6 +329,13 @@ def parse_devdoc(tag: str, devdoc: dict | None) -> dict:
     return natspecs
 
 
+class StorageField(NamedTuple):
+    label: str
+    data_type: str
+    slot: int
+    offset: int
```

Comment:
How expressive is this? Can it capture, for example, the case in which two consecutive slots share elements of a single `struct`? It's ok if not, this can be the next PR, I suppose.

Reply:
For example, there's a real-world case when
```
      {
        "astId": 1371,
        "contract": "contracts/C.sol:C",
        "label": "_x",
        "offset": 0,
        "slot": "1",
        "type": "t_struct(SomeType)3669_storage"
      },
      {
        "astId": 1374,
        "contract": "contracts/C.sol:C",
        "label": "_y",
        "offset": 0,
        "slot": "9",
        "type": "SomeOtherType"
      },
```
and that struct is spread throughout slots 1 to 8.

Reply:
Having seen the code below, I think that it should be expressive enough, but that the code currently doesn't account for it.

Reply:
Yes, I can add some unit tests. I haven't looked at `structs` examples yet.